new-frontiers-14/frontier-station-14
# Arachnid Silk Craft V. 1

## About the PR



Adds a silk based poncho, classic poncho and a cape as silkcraft recipes for Arachnids.

## Why / Balance


As I felt that the roster of what you could make with silk seemed under developed. This PR adds two silk based ponchos and a cloak to deepen that roster somewhat. It was a project to give me some direction in putting together a complete crafting grid item.


## How to test


- Be spooder.
- Have silk.
- Open crafting menu. 
- Craft a poncho, classic poncho and cloak. 
- Don and view from all angles.
- When cut each of the articles give less than the silk cloth value required for their crafting (equivalent 3 with 2 return). So there should be material loss.

I am open to feedback on designs and visual consistency. I did my best to maintain the grayscale pallet of silk while aiming for visual interest.

## Media


![ss+(2024-08-02+at+10 36 29)](https://github.com/user-attachments/assets/24a36358-61ff-4599-8d7c-2839c1d7e1f8)



## Breaking changes


Nothing I could imagine.

**Changelog**




add: 3 new webcraft clothing items
